### PR TITLE
LPS-130621 Page Audit cached issues

### PR DESCRIPTION
**Pull request by sandrodw3**
Hi @4lejandrito 😄
This includes all the front of https://issues.liferay.com/browse/LPS-130621. I would like to clarify with you how we should operate with the issues load date, so for now I consider that commit like WIP.

### CODE REVIEW

**Comment by sandrodw3** [comment]
I have updated the frontend part according to what we agreed this morning, so by my side everything is done so far. I can't test it properly until the backend part is ready, so I leave this on hold for now.

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/IssuesList.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ export default function IssuesList() {
 
 	const [percentage, setPercentage] = useState(0);
 
+	const localizedIssues = layoutReportsIssues
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/IssuesList.js:38]: Right, thanks! 🙂

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/IssuesList.js [hunk 2]
```diff
@@ -58,24 +59,36 @@ export default function IssuesList() {
...
-				<LoadingProgressBar percentage={percentage} />
-			) : localizedIssues ? (
-				<Issues
-					layoutReportsIssues={localizedIssues.issues}
-					successImage={successImage}
-				/>
-			) : (
-				<NoIssuesLoaded />
+		<>
+			{localizedIssues && (
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/IssuesList.js:63]: Mmmm you are right, that could happen 😛 I'll add `&& !loading` to the condition. Thanks!

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by CarlosBrichete** [comment]
ci:reevaluate:1137401_6185

**Comment by CarlosBrichete** [comment]
Just started reviewing :)

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
Thanks @CarlosBrichete!

**Comment by 4lejandrito** [comment]
ci:forward

**Pull request closed without merging**